std/experimental/allocator/building_blocks/bitmapped_block.d

```diff
@@ -843,6 +843,19 @@ struct BitmappedBlock(size_t theBlockSize, uint theAlignment = platformAlignment
     assert(h3.totalAllocation(64 * 4096 + 1) >= 65 * 4096);
 }
 
+// Test owns
+@system unittest
```

Comment:
I assume this is not safe because of some other PRs that need to complete the feature. Please don't forget to make a pass later to migrate all attributes into the unittest.